unikraft/app/kernel.go

```diff
@@ -0,0 +1,384 @@
+package app
```

Comment:
Can you pls double check all new files and include the copyright preamble (make sure to update year 2023)